protocol/v2/message/msg.go

```diff
@@ -3,8 +3,6 @@ package message
 import spectypes "github.com/bloxapp/ssv-spec/types"
 
 const (
-	// SSVDecidedMsgType extends spec msg type
-	SSVDecidedMsgType spectypes.MsgType = 3
```

Comment:
@nivBlox Should `SSVSyncMsgType` remain 4?

Reply:
keep it 4